services/nublado/values-stable.yaml

```diff
@@ -108,3 +108,52 @@ nublado:
     I79d2OwyjAqpf3SJKiBOpimK+1Vh/ZcKIPBVT+INfoe9XPcKnFV9csmpFXfgR5gP
     TwIDAQAB
     -----END PUBLIC KEY-----
+
+  jupyterhub_config: |
+    '''Runtime configuration for JupyterHub in the LSST environment.
+    '''
```

Comment:
Ah okay yes this makes sense.  We just don't use the newer imports here.  All good.